.github/workflows/rnx-build.yml

```diff
@@ -10,6 +10,10 @@ on:
         description: "Supported device types are `device`, `emulator`, `simulator`"
         required: true
         default: "simulator"
+      distribution:
```

Comment:
Looks like a few changes in this file snuck into the PR. I see 2 instances of ::set-output but the rest should probably go in a different PR.

Reply:
This is just a copy from rnx-kit. I can separate it out to a different PR though.

Reply:
Makes sense. If its intentional, than maybe update the PR description or something like that.